web-platform/website/src/styles/Toasts.ts

```diff
@@ -10,9 +10,15 @@ export const ToastStyle = {
     fontSize: FontSizeRaw.Small.fontSize,
     opacity: 1,
     left: "12rem",
-    top: "-2rem"
+    top: "-2rem",
   }),
   opacityZero: style({
-    opacity: "0"
-  })
-}
\ No newline at end of file
+    opacity: "0",
+  }),
+
+  productToast: style({
+    maxWidth: "unset",
```

Comment:
`maxWidth: "unset"` should be on the Container, but `height: "38px"` should be on the Alert (so that when an Alert is dismissed, the Container also disappears). i.e. these properties should be split up.

I suggest moving these properties to the `ToastComponent.tsx` file also, since this file is used for a different kind of Toast in the EditAccountPasswordView

Reply:
The Toast file is for all toasts; it's why I differentiate between account and product toasts.